src/reducers/index.js

```diff
@@ -45,6 +45,12 @@ const initialState = {
   // Height of the best chain of the network arrived from ws data
   height: 0,
   wallet: null,
+  // Metadata of tokens
+  tokenMetadata: {},
+  // Token list of uids that had errors when loading metadata
+  tokenMetadataErrors: [],
```

Comment:
Should it be a set or a dict with uids as keys and more information (e.g. latest retry time) as values?

Reply:
As we discussed privately we won't use this state for now, so I will keep the array and when we decide to show those errors we can improve them.